Monika After Story/game/script-topics.rpy

```diff
@@ -6940,6 +6940,28 @@ label monika_citizenship:
     m 2eka "But I want to make you happy, so...I'm going to do everything I can to keep bettering myself while I'm stuck here!"
     m 1eka "Thank you for listening to me vent, [player]."
     return
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_procrastination",category=['advice'],prompt="Procrastination",random=True))
+
+label monika_procrastination:
+    m 1euc "Hey [player], have you ever had something to do that you found really boring?"
+    m 3ekd "So rather than taking a lot of time to do it, you just keep putting it off?"
```

Comment:
```suggestion
    m 3ekd "So much so that rather than taking a lot of time to do it, you just keep putting it off?"
```

Reply:
(Opinions on this one?)

Reply:
I think the addition of "so much so" unnecessarily just extends the sentence. W/o it the sentence already functions fine.